flakytests/cylc-get-config/04-dummy-mode-output.t

```diff
@@ -19,22 +19,29 @@
 # And no duplication dummy outputs (GitHub #2064)
 . "$(dirname "$0")/test_header"
 
-set_test_number 6
+set_test_number 10
 
 install_suite "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
 
 run_ok "${TEST_NAME_BASE}-validate" \
     cylc validate --debug "${SUITE_NAME}"
 
-suite_run_fail "${TEST_NAME_BASE}-run-live" \
+# Live mode run: outputs not received, suite shuts down early without running baz
+suite_run_ok "${TEST_NAME_BASE}-run-live" \
```

Comment:
Shouldn't Cylc `exit 1` in this case?

Reply:
No, the outputs to trigger the remaining tasks are not generated in live mode (in dummy modes, we fake all registered outputs) so under SoD the downstream tasks are not generated and the suite finishes early (in a certain sense) but that's not an error - because the suite is doing exactly what the graph says to do.